cmd/k8s-cloudkms-plugin/main.go

```diff
@@ -81,28 +85,36 @@ func main() {
 		kms.BasePath = fmt.Sprintf("http://localhost:%d", *fakeKMSPort)
 	}
 
```

Comment:
If the user specifies `--key-suffix` but is not using the `v2` (that is `v1`) then nothing happens and the `--key-suffix` is being ignored, since it is not used in any way in the `v1` implementation. The use of `--key-suffix` with `v1` can indicate confusion and misconfiguration though. If it is preferred to abort the execution in this case, I'll add the validation step.

Reply:
Yea - we should abort.